torch/_inductor/codecache.py

```diff
@@ -1186,6 +1187,263 @@ def load(cls, kernel_name, source_code):
         return getattr(mod, kernel_name)
 
 
+def _cuda_compiler() -> str:
+    if cuda_env.nvcc_exist(config.cuda.cuda_cxx):
+        return config.cuda.cuda_cxx
+    if cuda_env.nvcc_exist(os.getenv("CUDACXX")):
+        return os.getenv("CUDACXX")
+    if cuda_env.nvcc_exist(os.getenv("CUDA_HOME")):
+        return os.path.join(os.getenv("CUDA_HOME"), "bin/nvcc")
+    return "nvcc"
+
+
+def _cutlass_include_paths() -> List[str]:
+    from torch.utils import cpp_extension
+
+    cutlass_path = os.path.join(cpp_extension._TORCH_PATH, "../third_party/cutlass")
+    return [
+        os.path.join(cutlass_path, "include"),
+        os.path.join(cutlass_path, "tools/library/include"),
+        os.path.join(cutlass_path, "tools/library/src"),
+        os.path.join(cutlass_path, "tools/util/include"),
+    ]
+
+
+def _cuda_lib_options() -> List[str]:
+    from torch.utils import cpp_extension
```

Comment:
Just in case: the other day I've found out that `cpp_extension` doesn't quite work internally (in my case: for inline compilation with `ninja`). Maybe for this particular use case it's fine, though.

Reply:
By internally, do you mean fbcode or something else? Could you help explain more?

Reply:
Yes, in `fbcode`. More context in this [comment](https://fburl.com/diff/b1xcu3e5). Maybe irrelevant for this PR, just a heads-up.